packages/snap/integration-test/client-request.test.ts

```diff
@@ -139,4 +139,71 @@ describe('OnClientRequestHandler', () => {
       stack: expect.anything(),
     });
   });
+
+  it('computes fee for valid PSBT', async () => {
+    const response = await snap.onClientRequest({
+      method: 'computeFee',
+      params: {
+        account: account.id,
+        transaction:
+          'cHNidP8BAI4CAAAAAAM1gwEAAAAAACJRIORP1Ndiq325lSC/jMG0RlhATHYmuuULfXgEHUM3u5i4AAAAAAAAAAAxai8AAUSx+i9Igg4HWdcpyagCs8mzuRCklgA7nRMkm69rAAAAAAAAAAAAAQACAAAAACp2AAAAAAAAFgAUgu3FEiFNy9ZR/zSpTo9nHREjrSoAAAAAAAAAAAA=',
+        scope: BtcScope.Regtest,
+      },
+    });
+
+    expect(response).toRespondWith([
+      {
+        type: FeeType.Priority,
+        asset: {
+          unit: CurrencyUnit.Regtest,
+          type: expect.any(String),
+          amount: expect.any(String),
```

Comment:
I think the amount here is deterministic and can be checked exactly because the account has been created in this suite and has a single UTXO. But I might be missing something 🤔 .